sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -2001,7 +1727,9 @@
                               "errorLabel": "",
                               "errors": "",
                               "key": "studyStartDate",
-                              "tags": [],
+                              "tags": [
+                                "execute-calculation"

```

Comment:
for my info, what are this tags

Reply:
The warnings and calculations are now centralized on the API. To make the calculations react faster and be displayed to the user in a quicker way, every time that a change in the form is detected it is calling the API to get the numbers.
This tag identifies which components should trigger the API call. If a change is in a component without this tag the API call is not executed.